### PR TITLE
minor changes, mostly sudo

### PR DESCRIPTION
**Pull request by DerekfromMadison**
added sudo to many of the commands

### CODE REVIEW

**Comment by ainsleys** [comment]
@levackt can you review? I'm not sure if sudo will be required for all users so just want to double check if that's best practice. Other changes for clarity are excellent, and very much appreciated @DerekfromMadison ! Thank you!

**Comment by levackt** [comment]
Thanks @DerekfromMadison !

On sudo, check out the [Docker Post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/), essentially adding your user to the docker group, then you can do without sudo.

File: README.md [hunk 1]
```diff
@@ -120,7 +119,12 @@ rustup install nightly
 rustup target add wasm32-unknown-unknown --toolchain nightly
 ```
 
-3. Install cargo generate
+3. If using linux, install build dependency
+```
+sudo apt install build-essential
+```
+
+4.. Install cargo generate
```
Comment (reuvenpo) [inline README.md]: I'd write something like:
>Run `cargo install cargo-generate`

File: README.md [hunk 2]
```diff
@@ -120,7 +119,12 @@ rustup install nightly
 rustup target add wasm32-unknown-unknown --toolchain nightly
 ```
 
-3. Install cargo generate
+3. If using linux, install build dependency
```
Comment (reuvenpo) [inline README.md]: >If using linux, install the standard build tools:

**Review by reuvenpo — CHANGES_REQUESTED** [review]

**Review by levackt — APPROVED** [review]

**Pull request merged**